examples/lock-app/linux/README.md

```diff
@@ -15,3 +9,57 @@ The application could be build in the same manner as `all-clusters-app`:
 ```
 ? scripts/examples/gn_build_example.sh examples/lock-app/linux out/lock-app chip_config_network_layer_ble=false
 ```
+
+# Named Pipe Interface
+
+This application accepts commands over named pipe. Pipe is located in
+`/tmp/chip_lock_app_fifo-<PID>`.
+
+## Command Format
+
+Commands interface accepts commands formatted as a JSON object. Structure of the
+object:
+
+| Key      | Type   | Value Description                                                                               | Optional |
+| -------- | ------ | ----------------------------------------------------------------------------------------------- | -------- |
+| `Cmd`    | String | Contains name of the command to execute                                                         | No       |
+| `Params` | Object | Contains parameters for the command. Could be omitted if the command does not accept parameters | Yes      |
+
+For example:
+
+-   `{ "Cmd": "SetDoorState", "Params": { "EndpointId": 1, "DoorState": 2 } }` -
```

Comment:
Why we need to use separate `Params` key to group parameters?  How is this better than just specify the parameter something like C function?

"Cmd": "SetDoorState",  "EndpointId": 1, "DoorState": 2

Reply:
It's not inherently 'better' or 'worse' than proposed approach, it's just a bit different. While we may waste a bit of space in the command string, the command becomes a bit more structured, allowing to introduce generic parsers a but easier.

Also, that allow us to have parameters that could have 'Cmd' (or, as you proposed, 'Name') as a key, making the command easier to read for human eye.

I'm open to either approach anyway, so if you think that this approach is no good, I can make it plain JSON.

Reply:
"Params" introduced one extra level of parsing which could be avoided. JSON is already structured format.
No preference between 'Name' and 'Cmd', we normally align with existing term to make is consistent.   

Those are nit comments, I will leave the final decision to you.

Reply:
Given where we are in the schedule, marking this as resolved. @Morozov-5F Please feel to follow up if you feel strongly.